File: Service_eHospital_DongNai_A/Implement/ztmp_BCVP_064DService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class ztmp_BCVP_064DService : BaseService<ztmp_BCVP_064D, Iztmp_BCVP_064DRepository>
    , Iztmp_BCVP_064DService
    {
    private readonly Iztmp_BCVP_064DRepository _ztmp_BCVP_064DRepository;
    public ztmp_BCVP_064DService(Iztmp_BCVP_064DRepository ztmp_BCVP_064DRepository) : base(ztmp_BCVP_064DRepository)
    {
    _ztmp_BCVP_064DRepository = ztmp_BCVP_064DRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/ztmp_BCVP_064D.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ztmp_BCVP_064D } from './ztmp_BCVP_064D.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ztmp_BCVP_064DService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'SoHoaDonVAT', 'TenBenhNhan', 'SoVaoVien', 'NgayHoaDon', 'GiaTriHoaDon', 'GiaTriHuy', 'GiaTriHoanTra', 'GhiChu', 'HuyHoaDon', 'HoanTra', 'Loai', 'nhanvienthutien', 'Date01', 'Date02', 'NguoiThuTien'];

    List: ztmp_BCVP_064D[] | undefined;
    ListFilter: ztmp_BCVP_064D[] | undefined;
    FormData!: ztmp_BCVP_064D;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ztmp_BCVP_064D";
    }
}

